try:
    print("name")
  #  print(10/2)
except NameError as e:
  print("Exceptio", e)
except:
  print("Something else went wrong")